qan/analyzer/mysql/mysql.go

```diff
@@ -178,7 +178,11 @@ func (m *MySQLAnalyzer) Stop() error {
 	// instance are not affected.
 	m.mrms.Remove(m.protoInstance.UUID, restartChan)
 
-	return a.Stop()
+	if nil != a {
```

Comment:
I think this should be moved to the first line after the function definition as
```go
func (m *MySQLAnalyzer) Stop() error {
    if m.analyzer == nil {
        return nil
    }
   .
   . 
   .
```

Reply:
Updated. Thanks.